docs/specs/authentication-flow-api-reference.md

```diff
@@ -219,7 +232,21 @@ When you are in this step of this flow, you will see a response like the followi
             "identification": "email"
           },
           {
-            "identification": "phone"
+            "identification": "phone",
+            "risk_assessment": {
```

Comment:
`risk_assessment` sounds a bit too general? Since we're here specifically for detecting bot ("I'm not a robot" confirmation), what about something more specific to avoid future confusion with other kind of "risks"? e.g. impossible travel.

e.g. `bot_risk` or `bot_detection`

Reply:
What about something like

```yaml
captcha:
  enabled: true
  # This list contains the challenge-based providers
  providers:
  - type: cloudflare
  # risk_assessment is now nested under captcha. We can still have risk_assessment in the root in the future.
  risk_assessment:
    enabled: true
    providers:
    - type: recaptchav3
```

Reply:
Sounds fair. (still... if we have a better name. Since "root" or not is not very easy for eye / still easy to confuse in documents, etc)

btw, why not...

```yaml
captcha:
  enabled: true
  # This list contains the challenge-based providers
  provider: cloudflare
  # risk_assessment is now nested under captcha. We can still have risk_assessment in the root in the future.
  risk_assessment:
    enabled: true
    provider: recaptchav3
```

Reply:
OK. let me revise to remove the array. So we only allow configuring one provider at a time.